.github/workflows/ci.yml

```diff
@@ -0,0 +1,75 @@
+name: Backpack CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    
+    - name: nvm install
+      shell: bash -l {0}
```

Comment:
What does this line do?

Reply:
That is a good question I need to ask @k0nserv as I took this from backpack-ios GH Actions

Reply:
Ah ok, yeah I'm sure it's there for a good reason, it just wasn't immediately obvious (in which case maybe we should pop a comment above it)

Reply:
See https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#using-a-specific-shell

Reply:
It says `bash` is the default though, so what does this line actually do that we don't already get built in?

Reply:
We add the `-l` which from official bash docs says

```
-l        Make bash act as if it had been invoked as a login shell (see
                 INVOCATION below).
```

Reply:
Why do we need to set this?

Reply:
Referring again to @k0nserv as he set this up in iOS - guessing there was some issues with setting it up? I was planning to chat to him about some of these things to understand more what they meant

Reply:
So tried removing it and it seems to fail, I think using the logged in parameter allows it to assume a state of a logged in shell so has access for tooling?

![Screenshot 2020-10-27 at 11 23 36](https://user-images.githubusercontent.com/8831547/97295198-e5604100-1846-11eb-882c-da9c0df1d4e9.png)

Reply:
Ah fair enough! Thanks for checking it